### PR TITLE
feat: increase/decrease approval

### PR DESCRIPTION
**Pull request by sokel**
Add increase/decrease token functions.

PS: 1 line affected by `fmt`

### CODE REVIEW

File: blockchain/api.go [hunk 1]
```diff
@@ -755,6 +763,52 @@ func NewStandardToken(address common.Address, opts *chainOpts) (TokenAPI, error)
...
 }
 
+func (api *StandardTokenApi) IncreaseApproval(ctx context.Context, key *ecdsa.PrivateKey, spender common.Address, value *big.Int) error {
+	opts := api.opts.getTxOpts(ctx, key, api.opts.gasLimit)
+	tx, err := api.tokenContract.IncreaseApproval(opts, spender, value)
+	if err != nil {
+		return err
+	}
+
+	client, err := api.opts.getClient()
```
Comment (sshaman1101) [inline blockchain/api.go]: You don't need to init client and check for error here, just use `api.client`.

File: blockchain/api.go [hunk 2]
```diff
@@ -91,6 +91,14 @@ type TokenAPI interface {
 	AllowanceOf(ctx context.Context, from, to common.Address) (*big.Int, error)
 	// TotalSupply - all amount of emitted token
 	TotalSupply(ctx context.Context) (*big.Int, error)
+	// IncreaseApproval increase the amount of tokens that an owner allowed to a spender
+	// approve should be called when current allowance = 0. To increment
```
Comment (sshaman1101) [inline blockchain/api.go]: What happens if I'll call this func with non-zero current approval?

File: blockchain/api.go [hunk 3]
```diff
@@ -91,6 +91,14 @@ type TokenAPI interface {
...
+	// allowed value is better to use this function to avoid 2 calls (and wait until the first transaction is mined)
+	IncreaseApproval(ctx context.Context, key *ecdsa.PrivateKey, spender common.Address, value *big.Int) error
+	// DecreaseApproval - decrease the amount of tokens that an owner allowed to a spender
+	// approve should be called when current allowance = 0. To decrement
```
Comment (sshaman1101) [inline blockchain/api.go]: Copy-paste?

File: blockchain/api.go (hunk 2 above)
Comment (sokel) [reply blockchain/api.go]: it should increased allowance to spender by sended value

**Pull request merged**